opae-libs/plugins/xfpga/common_int.h

```diff
@@ -73,12 +73,12 @@
 #define MAP_1G_HUGEPAGE (0x1e << MAP_HUGE_SHIFT) /* 2 ^ 0x1e = 1G */
```

Comment:
This file is a part of opae-libs, so it needs to be submitted to that repo.

Reply:
@anandaravuri, please be sure this file is removed from the pull request and submitted to opae-libs instead.